src/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -5,31 +5,91 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.IO.Tests
 {
-    public class NullCtorTests
+    public class NullTests
     {
+        [Theory]
+        [InlineData(20)]
+        public async static Task TestNullStream_Flush(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Flush();
+                await Stream.Null.FlushAsync();
+            }
+        }
+        
+        [Theory]
+        [InlineData(20)]
+        public static void TestNullStream_Dispose(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Dispose();
+            }
+        }
+        
+        [Theory]
+        [MemberData(nameof(NullStream_CopyToData))]
+        public async static Task TestNullStream_CopyTo(Stream destination, int bufferSize, CancellationToken cancellationToken)
+        {
+            Stream.Null.CopyTo(destination, bufferSize);
+            await Stream.Null.CopyToAsync(destination, bufferSize, cancellationToken);
```

Comment:
What are these tests verifying, other than that they don't throw an exception?  Couldn't we check, for example, that the destination's position didn't change by the copy, since nothing should have been copied?